main/ZsensorHCSR501.ino

```diff
@@ -33,6 +33,10 @@
 void setupHCSR501() {
   Log.notice(F("HCSR501 pin: %d" CR), HCSR501_GPIO);
   pinMode(HCSR501_GPIO, INPUT); // declare HC SR-501 GPIO as input
```

Comment:
Could you add also the macros to the corresponding config_ files ?

Reply:
I'm not sure what you're wanting done here. `HCSR501_GPIO` is already defined in the config_.h file.